src/components/Candidate.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import { url } from './url';
+import axios from 'axios';
+
+const Candidate = ({ candidateName, voteCount, id, getCandidates }) => {
+  const getVote = async (id) => {
+    try {
+      const vote = await axios.get(`${url}/vote`, {
+        params: {
+          id: id,
+        },
+      });
+      getCandidates();
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
+  const onClick = () => {
+    getVote(id);
+  };
+  return (
+    <Wrapper>
+      <CandidateName>{candidateName}</CandidateName>
+      <VoteCount>{voteCount}</VoteCount>
+      <VoteButton onClick={onClick}>투표</VoteButton>
```

Comment:
따로 onClick 함수를 선언할 필요 없이 이렇게 바로 넣어줄 수도 있다고 하네요
```suggestion
      <VoteButton onClick={()=>{getVote(id)}}>투표</VoteButton>
```

Reply:
맞네요! 이게 훨 깔끔할 것 같네요